.circleci/config.yml

```diff
@@ -269,11 +283,12 @@ workflows:
   version: 2
   build-test:
     jobs:
-      - runtest
+      # - runtest
       - deployment-checks: *release-branches
       - integration-tests-cocoapods: *release-tags-and-branches
       - integration-tests-swift-package-manager: *release-tags-and-branches
-      - integration-tests-carthage: *release-tags-and-branches
+      - integration-tests-carthage
+      # - integration-tests-carthage: *release-tags-and-branches
```

Comment:
will revert these changes before merging

Reply:
Don't forget to revert 🕺🏼